Demo/Demo/ViewModels/BaseViewModel.swift

```diff
@@ -257,6 +296,13 @@ class BaseViewModel: ObservableObject, PayPalWebCheckoutDelegate, CardDelegate {
         return CoreConfig(clientID: DemoSettings.clientID, accessToken: token, environment: DemoSettings.environment.paypalSDKEnvironment)
     }
 
+    func getNativeCheckoutClient() async throws -> PayPalClient {
+        guard let config = await getCoreConfig() else {
+            throw CoreSDKError(code: 0, domain: "Error initializing paypal webcheckout client", errorDescription: nil)
+        }
+        return PayPalClient(config: config, returnURL: "com.paypal.android://paypalpay")
```

Comment:
I don't think we need the return URL anymore on iOS.

Reply:
constructor for it needs it, it might be removed in next version